ROADMAP.md

```diff
@@ -0,0 +1,187 @@
+OpenMDAO 2020 Development Roadmap
+=================================
+
+Author: Justin S. Gray   
+Date: 2019-10-24
```

Comment:
should this date be changed to reflect that it's post-workshop?
(or an additional revision date)